packages/@lightningjs/ui-components/src/components/TextMagnifier/TextMagnifier.d.ts

```diff
@@ -0,0 +1,20 @@
+import Surface, { SurfaceStyle } from '../Surface';
+import { StylePartial } from '../../types/lui';
+
+type TextMagnifierStyle = SurfaceStyle & {
+  color: string;
+  gutterX: number;
+  gutterY: number;
+  h: number;
+  marginX: number;
+  radius: number | string;
+  textStyle: any;
```

Comment:
Should this be set to `TextBoxStyle`, or maybe `lng.textures.TextTexture.Settings | string` (which `textStyle` is set to in TextBox.style.d.ts)?

Reply:
Yup :)